docs/config/php.md

```diff
@@ -308,4 +308,27 @@ services:
       - docker-php-ext-enable memcached
 ```
 
+Following example uses [build steps](./../config/services.md#build-steps) to install the `sqlsrv` and `pdo_sqlsrv` php extensions required for connecting PHP to MSSQL database.
+
+```yaml
+services:
+  appserver:
+    type: php:7.2
+    overrides:
+      environment:
+        ACCEPT_EULA: "Y"
+    build_as_root:
+      - apt-get update -y
+      - apt-get install apt-transport-https -y
+      - curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+      - curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list > /etc/apt/sources.list.d/mssql-release.list
```

Comment:
Should this be 18.04? The microsoft documentation says to change it to 18.04 for Bionic.